pkgs/applications/misc/terminator/default.nix

```diff
@@ -1,24 +1,56 @@
-{ stdenv, fetchurl, python2, keybinder3, intltool, file, gtk3, gobject-introspection
-, libnotify, wrapGAppsHook, vte
+{ stdenv
+, fetchFromGitHub
+, python
+, keybinder3
+, intltool
+, file
+, gtk3
+, gobject-introspection
+, libnotify
+, wrapGAppsHook
+, vte
 }:
 
-python2.pkgs.buildPythonApplication rec {
+python.pkgs.buildPythonApplication rec {
   name = "terminator-${version}";
-  version = "1.91";
+  version = "1.92";
 
-  src = fetchurl {
-    url = "https://launchpad.net/terminator/gtk3/${version}/+download/${name}.tar.gz";
-    sha256 = "95f76e3c0253956d19ceab2f8da709a496f1b9cf9b1c5b8d3cd0b6da3cc7be69";
+  src = fetchFromGitHub {
+    owner = "gnome-terminator";
+    repo = "terminator";
+    rev = "bb24273eb40dc5eac97de74064488701fa40a743";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Good point! The PR was already merged though - should I open a new one for the fixup, or reopen this one?

Reply:
Ugh, missed this was merged. Since merged PRs cannot be reopened, I pushed the fixes in 64c09a3d5c1.

Reply:
@jtojnar : Thanks!